### PR TITLE
docs: Update advanced-usage.md

### PR DESCRIPTION
**Pull request by xiaotiandada**
**Description:**
Describe your changes.

**Related issue:**
Add link to the related issue.

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

File: docs/advanced-usage.md [hunk 1]
```diff
@@ -19,12 +19,13 @@ Ensure that `package-lock.json` is always committed, use `npm ci` instead of `np
 
 ### Yarn
 
-Ensure that `yarn.lock` is always committed, pass `--frozen-lockfile` to `yarn install` when installing packages.
+Ensure that `yarn.lock` is always committed, pass `--frozen-lockfile` or `--immutable` to `yarn install` when installing packages.
```
Comment (brcrista) [inline docs/advanced-usage.md:22]: What's the difference between these options?  Is there reason to prefer one or the other?

Comment (xiaotiandada) [reply docs/advanced-usage.md:22]: Hope more people test

Comment (brcrista) [reply docs/advanced-usage.md:22]: Thanks @xiaotiandada.  I say we just recommend `--immutable` then:

```suggestion
To ensure that `yarn.lock` is always committed, use `yarn install --immutable` when installing packages.
```

**Comment by xiaotiandada** [comment]
node12.x 14.x 16.x.  yarn1.22.18

image

**Pull request merged**